daemon/algod/api/algod.oas2.json

```diff
@@ -3972,6 +4045,74 @@
           }
         }
       }
+    },
+    "SimulationUnnamedGroupResources": {
+      "description": "If unnamed resource access is allowed, this is the subset of unnamed resources that were accessed by this group and could benefit from group resource sharing. In other words, there are no restrictions on where these resources must be placed in the transaction group. Also see property 'unnamed-resources-accessed' in SimulateTransactionResult to see unnamed resources accessed by each transaction which cannot benefit from group sharing; in contrast, these resources must be placed local to that transaction.",
```

Comment:
I'm not really advocated _exactly_ this change, but thought it would be more helpful to force myself to write doen a stab at what I'm about to recommend, rather than just handwave at what I'm thinking

```suggestion
    "SimulatedGroupSharableAccesses": {
      "description": "These are resources that were accessed by this group that would normally have caused failure, but were allowed in simulation. In order to make this group valid for actual submission, these resources can be made available by any transaction of the group, because all accesses were performed in programs that share group resources. In contrast, accesses that are performed by code that cannot group share are noted in the 'unnamed-resources-accessed' property of SimulateTransactionResult of the specific transaction that requires access to the resource.",
```

With the name changes, I'm trying to convey a few things. First "Simulation" becomes "Simulated" to make it obvious, just by saying it, that we're talking about a result object, not an input.

"GroupSharable" starts to explain the "could benefit from group resource sharing" idea.

"Accesses" again hints at what we've got here, going a bit further than just describing the "type" (a resource) and saying "these are things that were accessed".

I'm tempted to say the description should avoid the whole "In contract..." thing. Over time, that part will be less relevant. And when it does happen, I'm thinking that UI should direct users to that description, which would be the better place to describe it.

Reply:
I adopted as much as I could from this in e423c7d2f. The trouble is since we merged the old `SimulationUnnamedGroupResources` and `SimulationUnnamedResourceAssignment` into just `SimulateUnnamedResourcesAccessed`, this struct must explain how both group sharing vs txn local resources must be placed.

Here's what I landed on:

> If unnamed resource access is allowed, these are resources that were accessed by this group that would normally have caused failure, but were allowed in simulation. Depending on where this object is in the response, it either contains the unnamed resources that were accessed by this group and could benefit from group resource sharing (if this is a field in SimulateTransactionGroupResult), or the unnamed resources that were accessed by each transaction which cannot benefit from group sharing (if this is a field in SimulateTransactionResult). In order to make this group valid for actual submission, resources that can benefit from group sharing can be made available by any transaction of the group; otherwise, resources must be placed in the same transaction which accessed them.